tests/test_persisted_scale_property.py

```diff
@@ -0,0 +1,18 @@
+"Contains a collection of pytest tests that validates the persisted scale properties."
+import pytest
+
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestPersistedScaleProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the persisted scale properties."""
+
+    @pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+    def test_bool_property(self, persisted_scale):
```

Comment:
I'm sorry, I forgot to bring up NI's regression testing conventions: https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20934/Regression-Test-Conventions

Rename to something like `test__persisted_scale__get_bool_property__returns_value`